src/Templates/Public/scripts/default.js

```diff
@@ -409,7 +409,7 @@ function setupSteppers() {
                 btn = stepper.find('.bs-stepper-content .bs-stepper-pane.active button.step-submit');
             }
 
-            if (btn) {
+            if (btn && !isEnterKey(e)) {
```

Comment:
If the `&& !isEnterKey(e)` not set, it adding new line and execute `Next` button action